packages/vkui/src/hooks/useMediaQueries.ts

```diff
@@ -3,48 +3,24 @@ import { MEDIA_QUERIES, type MediaQueries } from '../lib/adaptivity';
 import { mediaQueryNull } from '../lib/browser';
 import { useDOM } from '../lib/dom';
 
-interface StoredMediaQueries {
-  /**
-   * Постоянная ссылка на окно. Это необходима, чтобы пересоздать медиа выражение если окно вдруг измениться.
-   *
-   * > `styleguide`
-   * >
-   * > В данный момент изменения окна существует в рамках песочницы. По сути мы храним ссылку на окно ради этого кейса
-   * > в песочнице.
-   */
-  window: Window | undefined;
-  /**
-   * Хранит созданные медиа выражения.
-   *
-   * Достаточно единожды создать экземпляры `MediaQueryList`, а дальше подписываться на события
-   * через метод `addEventListener(type, listener)`.
-   *
-   * {@link https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList}
-   */
-  mediaQueries: MediaQueries | null;
-}
-
-const storedMediaQueries: StoredMediaQueries = {
-  window: undefined,
-  mediaQueries: null,
-};
+const mediaQueriesCache = new Map<Window | undefined, MediaQueries>();
```

Comment:
```suggestion
const mediaQueriesCache = new WeakMap<Window | undefined, MediaQueries>();
```
может `WeakMap` заюзать?

у нас `window` идёт из `useDOM()`  -> `useContext()` -> `getDom()`, когда VKUI приложение заанмаунтится, то на `window` не должно остаться ссылки и тогда с `WeakMap` сборщик мусора должен будет очистить уже ненужные св-ва

с `Map` ссылка на `window` останется